### ISSUE TITLE
byte array does not work

### ISSUE THREAD

**Issue by 0x53A** [Zaid-Ajaj/Fable.Remoting#96]
My models look like this:

```f#
type Mesh = {
    Hash: byte array // 32 bytes

    Vertices: Vector3 array
    Indices: uint32 array

    Color: string
}

/// reference a mesh
type PlacedMeshRef = {
    Hash: byte array
    Placement : ObjectPlacement
}
```

when I send them from server to client I get an error:

![grafik](https://user-images.githubusercontent.com/4236651/48261835-a69a0c00-e420-11e8-849e-fd7821743473.png)

```
Error: "Cannot convert ["JString","MYiahJg85WmXkHFtQHhmaEoy0JvgZGZmqnE9X0/jvMs="] to ["Array",null]"
```

![grafik](https://user-images.githubusercontent.com/4236651/48261880-c7faf800-e420-11e8-8626-bb5d1f35d615.png)

It seems the issue is that the server encodes the byte array as base64, but the client expects an actual array.

Encoding it as base64 is probably a good idea to reduce data waste, so I guess the client should be updated to special case byte arrays.

**Comment by Zaid-Ajaj** [comment]
I didn't know the server does that haha, no problem I will try to fix this one today, of course, if you want to give it a try too then be my guest :smile: PR's are always welcome

**Comment by 0x53A** [comment]
I'll try to send _something_ tomorrow.

Looks like uint is also not supported:

![grafik](https://user-images.githubusercontent.com/4236651/48270212-c984ea00-e439-11e8-8cd1-cf9794404ac6.png)

```f#

type Mesh = {
    Hash: string //byte array // 32 bytes

    Vertices: V3 array
    Indices: uint32 array // <--- I guess it fails here

    Color: string
}
```


-------------

No uint

![grafik](https://user-images.githubusercontent.com/4236651/48270280-f6390180-e439-11e8-8b20-04cc837b3f95.png)

**Comment by Zaid-Ajaj** [comment]
Fixed in [SimpleJson/PR-18](https://github.com/Zaid-Ajaj/Fable.SimpleJson/pull/18), will patch remoting client as soon the nuget is ready

**Comment by Zaid-Ajaj** [comment]
Fix and published in version `4.5.3` of the remoting client, can you please update and check whether the issue persists? 

I will open a separate issue for unsigned integers
